subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/annotation/validation/NoSideEffectsRulesValidator.kt

```diff
@@ -0,0 +1,18 @@
+package com.avito.android.runner.annotation.validation
+
+import org.junit.rules.TestRule
+
+internal object NoSideEffectsRulesValidator {
```

Comment:
I don't find where will we validate test rules?

Reply:
In `InHouseInstrumentationTestRunner.onCreate`